composer.json

```diff
@@ -122,6 +118,7 @@
     "sensio/framework-extra-bundle": "^3.0.29",
     "sensio/generator-bundle": "^3.1.7",
     "shopsys/doctrine-orm": "2.6.2",
+    "shopsys/jparser": "0.1",
```

Comment:
This is a different use-case. We switched to `shopsys/jparser` library. And since none of our dependencies require the original library, there is no need for `replace`.

The use-case with doctrine is more complicated:
```
SSFW ---require---> joschi127/doctrine-entity-override-bundle ---require---> doctrine/orm
```
And we want to force composer to download only `shopsys/doctrine-orm` and NOT `doctrine/orm`. So we have to declare that `shopsys/doctrine-orm REPLACES doctrine/orm`.

In the end even `joschi127/doctrine-entity-override-bundle` requires `doctrine/orm`, this requirement will be satisfied by `shopsys/doctrine-orm`.

But as said previously, the use-case with `jparser` is not so complicated, so there is no need for the replace.